projects/plugins/jetpack/extensions/extended-blocks/core-embed/index.native.js

```diff
@@ -0,0 +1,4 @@
+export const reactivateFacebookEmbedBlockVariation = () => require( './facebook' );
+export const reactivateInstagramEmbedBlockVariation = () => require( './instagram' );
+export const registerLoomVariation = () => require( './loom' );
+export const registerSmartframeVariation = () => require( './smartframe' );
```

Comment:
My assumption here is that the first two constants are started with `reactivate` since these embed variants are part of the core but the last two utilize `register` because they are contained within Jetpack. Let me know if that was your thought process here.

Reply:
Exactly, Facebook ([reference](https://github.com/WordPress/gutenberg/blob/d64fd2befbadc733e51b9044860ad5a3db4df603/packages/block-library/src/embed/variations.js#L59-L73)) and [Instagram](https://github.com/WordPress/gutenberg/blob/d64fd2befbadc733e51b9044860ad5a3db4df603/packages/block-library/src/embed/variations.js#L75-L85) variations are already part of the core, that's why they reference the word `reactivate` and not `register` in the function.

For further information, in the following items you can see how the reactivation works for Facebook and Instagram variations in Jetpack:
https://github.com/Automattic/jetpack/blob/512f942dc5599b313797d6848f4572604fa887b8/projects/plugins/jetpack/extensions/extended-blocks/core-embed/facebook.js#L29-L67
https://github.com/Automattic/jetpack/blob/512f942dc5599b313797d6848f4572604fa887b8/projects/plugins/jetpack/extensions/extended-blocks/core-embed/instagram.js#L29-L67

Reply:
Ah, interesting. Thanks for providing the respective references. I will take a look.